README.md

```diff
@@ -63,41 +63,60 @@ For more background check [#62](https://github.com/js-reporters/js-reporters/iss
 
 ### Event Data
 
-Based on the discussion in [#12](https://github.com/js-reporters/js-reporters/issues/12#issuecomment-120483356), there are two basic data structures: Suites and Tests. A test represents an atomic test/spec/`it()`. A suite contains tests and optionally other suites.
+Based on the discussion in [#12](https://github.com/js-reporters/js-reporters/issues/12#issuecomment-120483356), there are two basic data structures: **Suites** and **Tests**. A test represents an atomic test/spec/`it()`. A suite contains tests and optionally other suites.
 
-For `testStart` and `testEnd`, the corresponding test object is passed to the reporter. The same applies to `suiteStart` and `suiteEnd` where the matching suite object is passed to the reporter. For `runStart` and `runEnd` a "global" suite object is passed to the reporter, this suite wraps all other top-level user defined suites as its child suites, it will be reffered to as `globalSuite`.
+The structures have been divided in two parts, a start part and an end part:
+ - SuiteStart
+ - SuiteEnd
+ - TestStart
+ - TestEnd
+
+For `testStart` and `testEnd`, the corresponding TestStart, respectively TestEnd, object is passed to the reporter. The same applies to `suiteStart` and `suiteEnd` where the matching SuiteStart/SuiteEnd object is passed to the reporter. For `runStart` and `runEnd` a "global" suite object is passed to the reporter, this suite wraps all other top-level user defined suites as its child suites, it will be reffered to as `globalSuite`.
 
 The data structures are defined as follows:
 
-- **Suite**: `Object` - A suite is a collection of tests and potentially other suites.
+- **SuiteStart**: `Object` - A SuiteStart is a collection of tests-starts and potentially other suites-starts, emitted before the suite have been executed, which means before executing any of its tests and child suites.
  - **name**: `String|undefined` - name of the suite, will be `undefined` only for the `globalSuite`.
  - **fullname**: `Array` - array of strings containing the name of the suite and the names of all its suites ancestors.
  - **tests**: `Array` - array containing all tests that directly belong to the suite (but not to a child suite).
  - **childSuites**: `Array` - array with all direct subsuites.
- - **status**: `String|undefined` - summarized status of the suite, it will be `undefined` on the `suiteStart` and `runStart` event.
+ - **testCounts**: `Object` - contains the total number of tests in the suite, including the tests of the child suites.
+    - **total**: `Number` - total number of tests
+
+- **SuiteEnd**: `Object` - A SuiteEnd is a collection of tests-ends and potentially other suites-ends, emitted after the suite has been executed, which means that all its tests and child suites have been also executed.
+ - **name**: `String|undefined` - name of the suite, will be `undefined` only for the `globalSuite`.
```

Comment:
How about removing the duplication here by saying something like "In addition to the properties of `SuiteStart`, `SuiteEnd` also has the following properties:"?